src/components/calcite-tab-nav/calcite-tab-nav.scss

```diff
@@ -1,50 +1,49 @@
 :host {
-  z-index: 2;
-  position: relative;
-  display: flex;
+  @apply z-20 relative flex;
 }
 
 .tab-nav {
-  display: flex;
-  width: 100%;
-  overflow: auto;
-  justify-content: flex-start;
-  -webkit-overflow-scrolling: touch;
+  @apply flex
+    w-full
+    overflow-auto
+    justify-start
+    scrolling-touch;
   // workaround for outset focus within a scrolling container
-  padding: 4px;
-  margin: -4px;
+  padding: theme("padding.1");
+  margin: theme("margin.-1");
```

Comment:
p-1 m--1 ?

Reply:
yeah I was trying to retain that comment directly above but I'll remove it and go with those instead 👍